docs/diagrams/datetimeutil/DateTimeUtilArchitecture.puml

```diff
@@ -0,0 +1,42 @@
+@startuml
+!include ../style.puml
+show members
+hide empty methods
+hide empty fields
+skinparam ClassFontColor #000000
+skinparam ClassBorderColor #000000
+
+title Date-Time Parser Partial Class Diagram
+
+class DateUtil {
```

Comment:
would you want to put the classes where u use static methods from like this instead

![image](https://user-images.githubusercontent.com/13099969/160565963-e9ad4b5f-98e6-4ad4-abf7-8ce3f6b28b7f.png)

as per the textbook https://nus-cs2103-ay2122s2.github.io/website/se-book-adapted/chapters/uml.html#calls-to-static-methods

just for your consideration, cos i did it on #251